packages/sdks-tests/src/e2e-tests/sdk.spec.ts

```diff
@@ -0,0 +1,20 @@
+import {  excludeGen2, test } from '../helpers/index.js';
+import { launchEmbedderAndWaitForSdk } from '../helpers/visual-editor.js';
+
+const SDK_INJECTED_MESSAGE =
+  'BUILDER_EVENT: builder.sdkInjected modelName: page apiKey: abcd';
+    
+test.describe('SDK', () => {
+  test('should inject correct SDK data into iframe', async ({ page, basePort, sdk }) => {
+    test.skip(excludeGen2(sdk));
+    const msgPromise = page.waitForEvent('console',  msg => msg.text().includes(SDK_INJECTED_MESSAGE));
```

Comment:
- This new test is not passing. Is it a feature implementation error, or an issue with how the test is written? From looking at your code in `embedded/index.js`, this should work.  
- `sdk.spec.ts` is too generic of a name for a test file. I would move this test case to `editing.spec.ts`, which covers all test cases tied to communicating with the Visual Editor